test/fixtures/currencies/currencies.js

```diff
@@ -7,9 +7,18 @@ const { DINERO, NEELAM } = require("../../../constants/wallets");
  * @return {Object}
  */
 
-const currencies = {
-  [DINERO]: 1000,
-  [NEELAM]: 2,
+module.exports = () => {
+  return [
+    {
+      [DINERO]: 1000,
```

Comment:
I did not understand this syntax why it is `[DINERO]` and not `"DINERO"`?

Reply:
Also, why did you export a function from here for any specific reason? Like you can just export an object it will be easy to consume as well. you need not to call it when using,

Reply:
![image](https://user-images.githubusercontent.com/63191499/162015179-1f399a71-feeb-4bb0-a24f-20ef1d014aa1.png)

DINERO is a variable, that's why it's like that. 

For second question. Yes, I was planning on doing that but most fixtures follow that pattern so continuing with function export. Should I change that?